package.json

```diff
@@ -105,7 +105,7 @@
     "sinon": "^2.1.0",
     "sinon-chai": "^2.8.0",
     "uglifyjs-webpack-plugin": "^1.1.4",
-    "url-loader": "^0.5.8",
+    "url-loader": "^1.1.2",
```

Comment:
I don't think we need to update any dependency, do we?

Reply:
Agree